vars/pushDockerfileEnvVarChangePR.groovy

```diff
@@ -41,13 +36,7 @@ def call(body) {
         flow.updateDockerfileEnvVar("${dockerfileName}", config.propertyName, config.version)
 
         container(name: containerName) {
-
-            sh 'chmod 600 /root/.ssh-git/ssh-key'
-            sh 'chmod 600 /root/.ssh-git/ssh-key.pub'
-            sh 'chmod 700 /root/.ssh-git'
-
-            sh "git config --global user.email fabric8-admin@googlegroups.com"
-            sh "git config --global user.name fabric8-release"
+            flow.setupGitSSH()
```

Comment:
may be worth moving `def flow` close to its usage.

Reply:
flow is used in several places in this file and hence defined at top.